modules/standard/Math.chpl

```diff
@@ -1196,7 +1196,15 @@ module Math {
     return a;
   }
 
-
+  /* Returns true if `x` and `y` are approximately equal, else returns false. */
+  inline proc isclose(x, y, rtol = 1e-5, atol = 0.0): bool {
```

Comment:
You should move this code to a separate branch and open a PR for it instead of putting it in a branch for `string.join`.